ui/views_test.py

```diff
@@ -23,7 +24,7 @@ def test_program_liveness(self):
         """Verify only 'live' program visible on homepage"""
         program_live_true = ProgramFactory.create(live=True)
         program_live_false = ProgramFactory.create(live=False)
-        response = self.client.get(reverse('ui-index'))
+        response = self.client.get('/')
```

Comment:
There's a urlpattern which takes in the entire path as an argument, so instead of `/` we would end up with `reverse('wagtail_serve', args=('',))`